base_local_planner/src/map_grid_cost_function.cpp

```diff
@@ -111,6 +112,11 @@ double MapGridCostFunction::scoreTrajectory(Trajectory &traj) {
       }
     }
 
+    if (firstCost < 0)
+    {
+      firstCost = grid_dist;
```

Comment:
My thought was to normalize by the current distance of the robot to the plan / goal (both use map_grid_cost_function's).  The first sample is always the robot's current pose, so I'm just recording it as I loop through everything.  I will change the variable name to make it more explicit.